packages/vue-query/test-setup.ts

```diff
@@ -0,0 +1,12 @@
+import { vi } from 'vitest'
```

Comment:
note: this is how I ended up mocking the `vue-demi` package in our tests (ended up replacing the file: `packages/vue-query/__mocks__/vue-demi.ts` as I didn't fully understand how these `__mocks__` directories are getting used.